src/receipt.rs

```diff
@@ -863,7 +871,7 @@ impl bytes::Deserializable for Receipt {
     fn from_bytes(bytes: &[u8]) -> io::Result<Self> {
         let mut instance = Self::ret(Default::default(), 0, 0, 0);
 
-        instance.write(bytes)?;
+        let _ = instance.write(bytes)?;
```

Comment:
Probably left from debugging?

```suggestion
        instance.write(bytes)?;
```

Reply:
Necesarry sacrifice to god Clippy.
If ommited it will report `error: written amount is not handled.`

Reply:
Is that indicative that it should be handled then?? 😂 What does `write` return? A `Result`?

Reply:
God Victor accused god Clippy of being inacurate and clippy was silenced. Victor blessings can be read here: https://github.com/FuelLabs/fuel-tx/pull/53#discussion_r778759630
it returns number of written bytes, in this particular case we are sure that all bytes are going to be written or error will be returned. Error handling part is here `)?;`

Reply:
Ah got it 😂. Can you document this in a comment? Otherwise in the future someone who's not aware of the context might think to delete the `let _ = ` and/or handle the error redundantly.

Reply:
sure, comment is added.